src/Clients/IMessageClient.cs

```diff
@@ -21,7 +21,7 @@ public interface IMessageClient
         /// Deletes a message.
         /// </summary>
         /// <remarks>https://getstream.io/chat/docs/dotnet-csharp/send_message/?language=csharp</remarks>
-        Task<GenericMessageResponse> DeleteMessageAsync(string messageId, bool hardDelete = false);
+        Task<GenericMessageResponse> DeleteMessageAsync(string messageId, string userId, bool hardDelete = false, bool deleteForMe = false);
```

Comment:
1. This would be a breaking change. It would be better to add this as an overload, so such a small change doesn't require customers to update their code. 
2. It's not clear what the `userId` means in the deletion context. Seems better to name it the same as API does: `deletedBy`